server/main.lua

```diff
@@ -16,6 +19,7 @@ local function AddCityJob(jobName, label)
         return false, "already added"
     else
         availableJobs[jobName] = label
+        TriggerClientEvent('qb-cityhall:Client:AddCityJob',-1)
```

Comment:
Missing whitespace before `-1`

Reply:
Whitespace is not needed.

Reply:
It's not a requirement, but it's good to adhere to a consistent code style

Reply:
In your above code

`RegisterNetEvent('qb-cityhall:Client:AddCityJob', function()
    getJobs()
end)`

vs 

`RegisterNetEvent('qb-cityhall:Client:AddCityJob',function()
    getJobs()
end)`